frontend/components/forms/fields/InputField/_styles.scss

```diff
@@ -114,21 +121,14 @@
 
   &__copied-confirmation {
     font-size: $x-small;
-    position: absolute;
     background-color: $ui-light-grey;
     border: solid 1px $ui-fleet-black-10;
     border-radius: $border-radius-xlarge;
     padding: $pad-xxsmall 6px;
-    top: 50%;
```

Comment:
Dev note: We can't use percentages like this because it doesn't play nice with taller InputField UI such as input fields with help text, or textareas (I refactored so this now works universally)